stream_alert_cli/manage_lambda/package.py

```diff
@@ -160,67 +146,89 @@ def _resolve_third_party(self, temp_package_path):
 class RuleProcessorPackage(LambdaPackage):
     """Deployment package class for the StreamAlert Rule Processor function"""
     config_key = 'rule_processor_config'
-    package_folders = {
-        'stream_alert/rule_processor', 'stream_alert/shared', 'rules', 'matchers', 'helpers', 'conf'
+    lambda_handler = 'stream_alert.rule_processor.main.handler'
+    package_files = {
```

Comment:
Does it make sense to create a set with common package files between all functions?

Reply:
I just checked - there actually isn't a single file used by all functions. The apps only use a single folder (`app_integrations`)! We could create a common set for the rest of the functions, but I personally am in favor of the explicit list for each package - it makes it very easy to read